packages/context/src/api/paramsSerializer.ts

```diff
@@ -1,11 +1,25 @@
 import { ParamsSerializerOptions } from 'axios';
+import _ from 'lodash';
 
 export function paramsSerializer(params: Record<string, unknown>, options?: ParamsSerializerOptions) {
     const searchParams = new URLSearchParams();
 
     for (const [key, value] of Object.entries(params)) {
+        if (!validateParam(key) || !validateParam(value)) {
+            continue;
+        }
+
         searchParams.append(key, typeof value !== 'string' ? JSON.stringify(value) : value);
     }
 
     return searchParams.toString();
 }
+
+function validateParam(item: unknown) {
+    return (
+        !_.isNil(item) &&
+        !_.isUndefined(item) &&
+        !(_.isObject(item) && _.isEmpty(item)) &&
+        !(_.isString(item) && _.isEmpty(item))
```

Comment:
Why are empty objects and empty strings not valid?  Empty strings are definitely valid and a common occurrence.  The only thing that is clear to me that shouldn't be sent is `undefined`.  Anything else seems like a legitimate value.